lib/src/api/method/mod.rs

```diff
@@ -692,11 +693,10 @@ where
 	/// # Ok(())
 	/// # }
 	/// ```
-	pub fn select<R>(&self, resource: impl opt::IntoResource<R>) -> Select<C, R> {
+	pub fn select<O>(&self, resource: impl IntoResource<O>) -> Select<C, O> {
```

Comment:
What's the `O` here for, output?

Reply:
Yes, I can change it back to R it was changed when I experimented with adding an extra type for the type or resource.

Reply:
Unless there are other changes you will need to do for the PR, we can just leave it as is for now since changing it back is not a breaking change. It's not necessary to dismiss current reviews just for this.